azurerm/internal/services/keyvault/key_vault_access_policy_resource.go

```diff
@@ -327,3 +351,25 @@ func FindKeyVaultAccessPolicy(policies *[]keyvault.AccessPolicyEntry, objectId s
 
 	return nil, nil
 }
+
+func accessPolicyRefreshFunc(ctx context.Context, client *keyvault.VaultsClient, resourceGroup string, vaultName string, objectId string, applicationId string) resource.StateRefreshFunc {
+	return func() (interface{}, string, error) {
+		log.Printf("[DEBUG] Checking for completion of Access Policy create/update")
+
+		read, err := client.Get(ctx, resourceGroup, vaultName)
+		if err != nil {
+			if utils.ResponseWasNotFound(read.Response) {
+				return "vaultnotfound", "vaultnotfound", fmt.Errorf("failed to find vault %q (resource group %q)", vaultName, resourceGroup)
+			}
+		}
+
+		if read.Properties != nil && read.Properties.AccessPolicies != nil {
+			policy, _ := FindKeyVaultAccessPolicy(read.Properties.AccessPolicies, objectId, applicationId)
```

Comment:
this should still check for the error here

Reply:
err is always `nil` for `FindKeyVaultAccessPolicy`, not picked up by the linter. Fix the func?